code/obj/item.dm

```diff
@@ -286,8 +286,15 @@
 
 /obj/item/New()
 	// this is dumb but it won't let me initialize vars to image() for some reason
+
+	// this is here so things wont be dumb
+	var/wstate = icon_state
+	if (wear_state)
+		wstate = wear_state
+
 	wear_image = image(wear_image_icon)
-	wear_image.icon_state = icon_state //Why was this null until someone actually wore it? Made manipulation impossible.
+	wear_image.icon_state = wstate //Why was this null until someone actually wore it? Made manipulation impossible.
+
```

Comment:
```suggestion
	wear_image.icon_state = icon_state || wear_state
```
This makes it so the `wear_image` uses `icon_state` if it's set and `wear_state` if it isn't, same as above but in way less lines.